cmd/node/config/gasSchedules/gasScheduleV2.toml

```diff
@@ -1,7 +1,7 @@
 [BuiltInCost]
     ChangeOwnerAddress    = 5000000
     ClaimDeveloperRewards = 5000000
-    SaveUserName          = 5000000
+    SaveUserName          = 1000000
```

Comment:
5x cheaper?

Reply:
@sasurobert ?

Reply:
yes. it is a simple save which will come through the DNS contract which already consumes around 14-15 million gas. plus async call gas.